chess_gui.py

```diff
@@ -1,271 +1,337 @@
-    #                             valid_moves = []
-    #                         else:
-    #                             game_state.move_piece((player_clicks[0][0], player_clicks[0][1]),
-    #                                                   (player_clicks[1][0], player_clicks[1][1]), False)
-    #                             square_selected = ()
-    #                             player_clicks = []
-    #                             valid_moves = []
-    #
-    #                             ai_move = ai.minimax(game_state, 3, -100000, 100000, True, Player.PLAYER_2)
-    #                             game_state.move_piece(ai_move[0], ai_move[1], True)
-    #                     else:
-    #                         valid_moves = game_state.get_valid_moves((row, col))
-    #                         if valid_moves is None:
-    #                             valid_moves = []
-    #             elif e.type == py.KEYDOWN:
-    #                 if e.key == py.K_r:
-    #                     game_over = False
-    #                     game_state = chess_engine.game_state()
-    #                     valid_moves = []
-    #                     square_selected = ()
-    #                     player_clicks = []
-    #                     valid_moves = []
-    #                 elif e.key == py.K_u:
-    #                     game_state.undo_move()
-    #                     print(len(game_state.move_log))
-    #         draw_game_state(screen, game_state, valid_moves, square_selected)
-    #
-    #         endgame = game_state.checkmate_stalemate_checker()
-    #         if endgame == 0:
-    #             game_over = True
-    #             draw_text(screen, "Black wins.")
-    #         elif endgame == 1:
-    #             game_over = True
-    #             draw_text(screen, "White wins.")
-    #         elif endgame == 2:
-    #             game_over = True
-    #             draw_text(screen, "Stalemate.")
-    #
-    #         clock.tick(MAX_FPS)
-    #         py.display.flip()
-    #
-    # elif human_player is 'b':
-    #     pass
-
-
-def draw_text(screen, text):
-    font = py.font.SysFont("Helvitca", 32, True, False)
-    text_object = font.render(text, False, py.Color("Black"))
-    text_location = py.Rect(0, 0, WIDTH, HEIGHT).move(WIDTH / 2 - text_object.get_width() / 2,
-                                                      HEIGHT / 2 - text_object.get_height() / 2)
-    screen.blit(text_object, text_location)
-
-
-if __name__ == "__main__":
-    main()
+#
+# The GUI engine for Python Chess
+#
+# Author: Boo Sung Kim, Eddie Sharick
+# Note: The pygame tutorial by Eddie Sharick was used for the GUI engine. The GUI code was altered by Boo Sung Kim to
+# fit in with the rest of the project.
+#
+import chess_engine
+import pygame as py
+import logging
+from Piece import Piece
+
+import ai_engine
+from enums import Player
+logger = logging.getLogger(__name__)
+
+def display_board(board):

```

Comment:
That's a bad displaying of the board.
Look at the logs and try to make it better and more comprehensible.